Makefile

```diff
@@ -5,12 +5,12 @@ project := xlang
 install:
 	glide --version || go get github.com/Masterminds/glide
 	GO15VENDOREXPERIMENT=1 glide install
-	GO15VENDOREXPERIMENT=1 go build `GO15VENDOREXPERIMENT=1 glide novendor`
+	GO15VENDOREXPERIMENT=1 go build `glide novendor`
```

Comment:
You can do top level exports,
https://github.com/uber/tchannel-go/blob/dev/Makefile#L12

Each command for a target runs in its own sub-shell so having an `env` target won't work (it's lost after that shell). The export command is what you want to use.

Then you don't need to keep setting the variable everywhere.